superset/assets/javascripts/modules/superset.js

```diff
@@ -207,11 +203,16 @@ const px = function (state) {
         } else {
           this.force = force;
         }
+        const formDataExtra = Object.assign({}, formData);
+        const extraFilters = controller.effectiveExtraFilters(sliceId);
+        formDataExtra.filters = formDataExtra.filters.concat(extraFilters);
+        controls.find('a.exploreChart').attr('href', getExploreUrl(formDataExtra));
```

Comment:
Does not look a very react-ish way to handle this, shouldn't the component that renders the link should take formDataExtra and render again when it changes?

Reply:
@xrmx  eventually this whole module will get refactored out as we write a 100% React dashboard app (currently we have a lot of non-react JS in that app...), this is ok as is for now. We'll have to nuke all the jQuery code out of there in the future...